### ISSUE TITLE
The hoursBetweenBackups of CosmosDB Cassandra Cluster is deprecated

### ISSUE THREAD

**Issue by neil-yechenwei** [Azure/azure-rest-api-specs#24074]
Previously we can set the “hoursBetweenBackups” property successfully and service API would return the corresponding value for this property. But recently though service API accepts “hoursBetweenBackups” while creating/updating but service API wouldn’t return this property anymore. Then I checked with service team and they replied "_To improve our backup service, we have introduced a new property called "backupSchedules" that allows you to define cronExpression ( when to start backups) and retentionInHours (how long to keep them). As a result, the previous parameter "hoursBetweenBackups" has been deprecated._" (Note: Currently "backupSchedules" isn't in azure rest api spec and it will be released to azure rest api spec in near future.). So I filed this issue for tracking.

**Comment by tombuildsstuff** [comment]
@neil-yechenwei just to confirm, is this a breaking change in the existing API version, or in a new one?

**Comment by neil-yechenwei** [comment]
Yes. It's a breaking change/regression bug. As service team shouldn't deprecate the field using this kind of way, thus it would cause breaking change/regression bug to the API. So I assume service team/ARM team has to fix/revert this change first and then remove it from azure-rest-api-spec and introduce new property in an entirely new API Version to deprecate this property.
